hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/AbstractRpcClient.java

```diff
@@ -380,7 +380,8 @@ private void onCallFinished(Call call, HBaseRpcController hrc, Address addr,
     }
     if (LOG.isTraceEnabled()) {
       LOG.trace(
-        "Call: " + call.md.getName() + ", callTime: " + call.callStats.getCallTimeMs() + "ms");
+        "CallId: {}, call: {}, startTime: {}ms, callTime: {}ms", call.id, call.getStartTime(),
```

Comment:
The order of the parameters is wrong? call.md.getName should be placed before call.getStartTime

Reply:
Sorry, i will fix it.